automation/taskcluster/lib/tasks.py

```diff
@@ -468,8 +471,8 @@ def _craft_apk_full_path_from_variant(variant):
     )
 
     short_variant = variant[:-len(build_type)]
-    postfix = '-unsigned' if build_type == 'release' else ''
-    product = '{}{}'.format(product[0].lower(), product[1:])
+    postfix = '-unsigned' if build_type.startswith('release') else ''
+    product = lower_case_first_letter(product)
```

Comment:
[Aren't products already all first-letter-lowercased?](https://github.com/mozilla-mobile/reference-browser/pull/703/files#diff-59f90d509bc6ae2c9d877250c5ba49c5R489)

Reply:
True. They are in r-b, but not in Fenix, for instance :/ It depends on how the gradle target are defined. 

/me wishes we had a single library